package.json

```diff
@@ -15,9 +15,9 @@
   ],
   "scripts": {
     "test": "npm run jshint && npm run test-unit && npm run test-e2e",
-    "test-unit": "jasmine-node --captureExceptions spec/unit",
-    "test-e2e": "jasmine-node --captureExceptions spec/e2e",
-    "cover": "istanbul cover --root template --print detail node_modules/jasmine-node/bin/jasmine-node -- spec/unit",
+    "test-unit": "jasmine --captureExceptions ",
```

Comment:
Unfortunately the option used here is not available in `jasmine` which caused these tests to not be run on Windows any more:

```
> cordova-windows@5.1.0-dev test-e2e C:\projects\cordova-windows
> jasmine --captureExceptions spec/e2e/endtoend.spec.js
Unknown options: --captureExceptions
Usage: jasmine [command] [options] [files]
Commands:
      init	initialize jasmine
  examples	install examples
   help,-h	show help
version,-v	show jasmine and jasmine-core versions
If no command is given, jasmine specs will be run
Options:
        --no-color	turn off color in spec output
         --filter=	filter specs to run only those that match the given string
         --helper=	load helper files that match the given string
--stop-on-failure=	[true|false] stop spec execution on expectation failure
         --config=	path to your optional jasmine.json
The given arguments take precedence over options in your jasmine.json
The path to your optional jasmine.json can also be configured by setting the JASMINE_CONFIG_PATH environment variable
Build success
```

Example: https://ci.appveyor.com/project/ApacheSoftwareFoundation/cordova-windows/build/1.0.404/job/61k4u344jus9t15l

Reply:
Turns out back when this was commited it probably did work, only later `jasmine` versions removed it.